components/camel-webhook/src/main/java/org/apache/camel/component/webhook/WebhookEndpoint.java

```diff
@@ -78,10 +79,16 @@ public void afterPropertiesConfigured(CamelContext camelContext) {
         delegateEndpoint.setWebhookConfiguration(configuration);
     }
 
+    @Override
+    protected void doInit() throws Exception {
+        super.doInit();
+        ServiceHelper.initService(delegateEndpoint);
+    }
+
     @Override
     protected void doStart() throws Exception {
         super.doStart();
-
+        ServiceHelper.startService(delegateEndpoint);
```

Comment:
if the delegateEndpoint has been stared as this router is added after the camel context started, what would happen ? The endpoint can be started twice ?

Reply:
Thats taken care of internally by Camel to check its state and only start if not already started etc.